project/ProjectPlugin.scala

```diff
@@ -156,6 +156,22 @@ object ProjectPlugin extends AutoPlugin {
       )
     )
 
+    lazy val exampleTodolistRuntimeSettings: Seq[Def.Setting[_]] = Seq(
+      libraryDependencies ++= Seq(
+        %%("monix")
```

Comment:
We are not doing streaming in this example, we can remove this dependency

Reply:
I need the dependency just for the `Scheduler` implicit requirement.